addons/sale/data/mail_template_data.xml

```diff
@@ -273,7 +273,7 @@
         </t>
         <t t-else="">
             has been confirmed.
-            <t t-if="object.currency_id.compare_amounts(object.amount_paid + transaction_sudo.amount, object.amount_total) >= 0">
+            <t t-if="not object.currency_id.compare_amounts(object.amount_paid + transaction_sudo.amount, object.amount_total) >= 0">
```

Comment:
I wonder if the `transaction_sudo.amount` needs to be taken into account in the condition here :thinking: because if the transaction went through, then the `object.amount_paid` is already increased by the amount of the transaction (and the amount of the transaction is not taken into account in the message below.
Also, I still prefer without the negation.
```suggestion
            <t t-if="object.currency_id.compare_amounts(object.amount_paid, object.amount_total) &lt; 0">
```

Reply:
Indeed, the `transaction_sudo.amount` is already added to the `object.amount_paid` so it shouldn't be taken into account